packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -137,6 +137,8 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, Acco
 
   private focusHandler: ContainerFocusHandler = null
   private itemRefs = []
+  private defaultAccordionTitleId = _.uniqueId('accordion-title-')
```

Comment:
nit `private readonly`, although I'm aware how @layershifter feels about modifiers 😆

Reply:
Now imagine you have 10 accordions on a page each having 10 sections :-)

Reply:
Visibility modifiers will be removed by lint rule on next week, no more discussions and comments around them